ipatests/test_integration/test_installation.py

```diff
@@ -392,6 +393,20 @@ def get_pki_tomcatd_pid(host):
     return(pid)
 
 
+def get_ipa_services_pids(host):
+    ipa_services_name = [
+        "krb5kdc", "kadmin", "named", "httpd", "ipa-custodia",
+        "pki_tomcatd", "ipa-dnskeysyncd"
```

Comment:
Why don't you use the same list as defined line 472?

Reply:
hello @flo-renaud , basically, there are two types of names list are present here, one showed and use by ipactl status command and the other one is the mapping done on the service names for fetching there original names from the system they are running on, that is why i have given two list of services,